DuckDuckGo/UnifiedFeedbackForm/UnifiedFeedbackSender.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  UnifiedFeedbackSender.swift
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import PixelKit
+
+protocol UnifiedFeedbackSender {
+    func sendFeatureRequestPixel(description: String, source: String) async throws
+    func sendGeneralFeedbackPixel(description: String, source: String) async throws
```

Comment:
I'm tempted to suggest changing `source` to some kind of enum, assuming that the source will always be one of a small set of options. Don't really feel that strongly though, I just prefer strong types for anything that isn't dynamic input.